### DISCUSSION TITLE
Distract Mod idea

### DISCUSSION THREAD

**Discussion by LIII-Works** [ppy/osu#13077] [Ideas]
**Describe the new feature:** 
inspired by [Aleph-0](https://osu.ppy.sh/beatmapsets/791900#osu)
Spawn fake circles to deceive player to increase the difficulty of reading
eg.
![image](https://user-images.githubusercontent.com/56265123/68987924-37909800-086a-11ea-886d-280226269049.png)
Green circle are real ,yellow,blue,purple are fake

~maybe this can be ranked~

**Proposal designs of the feature:** 
- Spawn same size and same color but semi lucent or special texture circles

- The number of fake circles should be determined by object number

- The animation of disappear of fake circle can be like this:
![red circle](https://user-images.githubusercontent.com/56265123/68988022-de296880-086b-11ea-84c6-c3750e6717fc.gif)

- If player hit wrong circle,decrease HP

- will not reset combo

-This picture shows where the wrong circles should spawn
[spawn.zip](https://github.com/ppy/osu/files/3853814/spawn.zip)

**Comment by AnxietyWump** [comment]
this is actually a decent idea. seems really cool!

**Comment by Tanza3D** [comment]
agreed, would love to see this in the game, as well as seeing it be ranked!
